misc_docs/syntax/language_labeled_argument.mdx

```diff
@@ -0,0 +1,36 @@
+---
+id: "labeled-argument"
+keywords: ["labeled", "argument"]
+name: "~arg"
+summary: "This is a `labeled argument`."
+category: "languageconstructs"
+---
+
+When declaring a function, arguments can be prefixed with `~` which means that they can and need to be called by their name, not the argument position. This is especially useful to differentiate them more easily if they are of the same type.
```

Comment:
Maybe mention explicitly (and add an example) that they can now be supplied in any order? And that it still works with partial application.